Cargo.toml

```diff
@@ -148,8 +148,11 @@ default-members = [
 ]
 
 [profile.release]
+opt-level = 3
 debug = true
-overflow-checks = true
+overflow-checks = false
```

Comment:
@msmouse ^^

Reply:
The cost of this flag is about 7% in performance on forge.
I can add a separate [profile.performance] which will differ from release by the value of overflow-checks.

Reply:
[source](https://doc.rust-lang.org/cargo/reference/profiles.html#dev)
The overflow-checks setting controls the [-C overflow-checks flag](https://doc.rust-lang.org/rustc/codegen-options/index.html#overflow-checks) which controls the behavior of [runtime integer overflow](https://doc.rust-lang.org/reference/expressions/operator-expr.html#overflow). When overflow-checks are enabled, a panic will occur on overflow.